transport/tchannel/config.go

```diff
@@ -23,6 +23,8 @@ package tchannel
 import (
 	"fmt"
 
+	opentracing "github.com/opentracing/opentracing-go"
+
```

Comment:
import group ordering should be,

- standard library
- local library imports
- other libraries